src/main.ts

```diff
@@ -3,25 +3,91 @@ import * as Tone from 'tone';
 
 class TestScene extends Phaser.Scene {
     synth: Tone.Synth
-
+    w!: number
+    h!: number
+    playBtn!: Phaser.GameObjects.Triangle
+    circles!: Phaser.GameObjects.Arc[]
     // gaming
     constructor() {
         super('test');
 
         this.synth = new Tone.Synth().toDestination()
-        this.synth.sync()
-        this.synth.triggerAttackRelease("C4", "8n", 0)
-        this.synth.triggerAttackRelease("E4", "8n", "64n")
-        this.synth.triggerAttackRelease("G4", "8n", "32n")
     }
+    makeTrack() {
+        this.add.rectangle(0, this.h * 0.2, this.w * 0.8, this.h * 0.2, 0xabcdef).setOrigin(0).setStrokeStyle(5, 0)
+        this.add.rectangle(0, this.h * 0.4, this.w * 0.8, this.h * 0.2, 0xabcdef).setOrigin(0).setStrokeStyle(5, 0)
+        this.add.rectangle(0, this.h * 0.6, this.w * 0.8, this.h * 0.2, 0xabcdef).setOrigin(0).setStrokeStyle(5, 0)
+        this.add.rectangle(0, this.h * 0.8, this.w * 0.8, this.h * 0.2, 0xabcdef).setOrigin(0).setStrokeStyle(5, 0)
+    }
+    setUp() {
+        this.w = this.game.config.width as number
+        this.h = this.game.config.height as number
+        this.makeTrack()
+        this.sound.on(Phaser.Sound.Events.UNLOCKED, () => Tone.start())
+
+        this.playBtn = this.add.triangle(this.w * 0.3, this.h * 0.1, 0, 0, 0, 100, 100, 50, 0x34ff34).setOrigin(0, 0.5).setInteractive({ useHandCursor: true })
+        this.circles = [
+            this.add.circle(this.w * 0.4, this.h * 0.1, 10, 0xffffff).setOrigin(0.5),
+            this.add.circle(this.w * 0.4 + 100, this.h * 0.1, 10, 0xffffff).setOrigin(0.5),
+            this.add.circle(this.w * 0.4 + 200, this.h * 0.1, 10, 0xffffff).setOrigin(0.5),
+            this.add.circle(this.w * 0.4 + 300, this.h * 0.1, 10, 0xffffff).setOrigin(0.5),
+        ]
+
+        Tone.Transport.stop()
+        Tone.Transport.start()
+
+        this.playBtn.on(Phaser.Input.Events.POINTER_DOWN, () => {
+            this.playBtn.setAlpha(0.5)
+            this.playBtn.disableInteractive()
+            let i = 0;
+            this.time.addEvent({
+                delay: 1000,
+                callback: () => {
+                    this.circles[i].setAlpha(0.5)
+
+                    this.synth.sync()
+                    if (i == 3) {
+                        this.synth.triggerAttackRelease("C5", "8n")
+                    } else {
+                        this.synth.triggerAttackRelease("C4", "8n")
+                    }
+                    i++
```

Comment:
This is still incorrect:

`sync` and `triggerAttackRelease` should only be done once: this puts the notes on the Transport (think of the Transport as a piece of sheet music)

`Transport.start()` and `Transport.stop()` control the musician playing the sheet music, and _these_ are what you should be calling repeatedly